### DISCUSSION TITLE
2022-03-07 Core team update

### DISCUSSION THREAD

**Discussion by mm-near** [near/community#225] [Core]
# TL;DR

* concrete improvements in rocksdb (due to compression change)
* first results (and Action Items) from multiple loadtests
* debug 'pages' are being submitted - should make it easier to debug mainnet/testnet issues

# Done

@pompon0 - Finished and submitted the chain fetching loadtest tool.
@pompon0 - Performed the loadtest, written down the report.

@mzhangmzz - Merged PR with the first iteration with mock network stress testing https://github.com/near/nearcore/pull/6340
@mzhangmzz Working on making start_mock_network binary work on mainnet data, still WIP https://github.com/near/nearcore/pull/6396

@EdvardD - Found & fixed regression in rocksdb 0.18 (and successfully migrated)
@EdvardD - Deployed lz4+zstd compression (should save around 20% of storage)
@EdvardD - Design doc for the 'cold' storage for archival node

@mm-near - loadtesting localnet (single node) -  submitted (https://github.com/near/nearcore/pull/6374)
@mm-near - debug page to see the detailed performance of the recent blocks - submitted (https://github.com/near/nearcore/pull/6354)
@mm-near - document with deep analysis of what went wrong during loadtest (and what we should do to fix it)
